resources/scripts/ttx_diff.py

```diff
@@ -347,7 +347,23 @@ def allow_some_off_by_ones(
         )
 
 
+# the order of lookups in a feature's lookuplist do not matter;
+# fontc always has them in sorted order but fontmake doesn't, so sort them
+def sort_fontmake_feature_lookups(ttx):
+    gpos = ttx.find("GPOS")
+    if not gpos:
+        return
+    features = gpos.xpath("//Feature")
+
+    for feature in features:
+        items = list(feature.iter())
+        # the first item is 'Feature', the second always a comment
```

Comment:
relying on a particular ordering of the elements is a bit brittle; how about collecting the list of all the LookupListIndex elements in this Feature and only then read/write their "value" attrib?

Reply:
that does seem better, yes :)